api/IDBCursor.json

```diff
@@ -673,10 +673,10 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "7"
+              "version_added": "10.1"
```

Comment:
Is it really the case that worker support for `IDBCursorWithValue` was in Safari 10, but `IDBCursor` came in 10.1? That's very strange given that `IDBCursorWithValue` inherits from `IDBCursor`...

Reply:
This is just updating its availability in workers!  The API is still marked as supported since Safari 6.1.

Reply:
Yes, but all the other worker support entries say Safari 10. It seems implausible that just this would be exposed later. Maybe a mishap with the interface object exposure, but the functionality itself, wasn't it there in 10?

Reply:
Ah, gotcha -- yeah, this is actually in Safari 10 so I feel this might have been a typo.  Fixed!